code/datums/status_effects/buffs.dm

```diff
@@ -597,3 +597,22 @@
 
 /datum/status_effect/jump_jet/on_remove()
 	owner.RemoveElement(/datum/element/forced_gravity, 0)
+
+/// Makes the mob immune to radiation for a short bit to help with safely spawning in hazardous areas
+/datum/status_effect/radiation_immunity
+	id = "radiation_immunity"
+	duration = 1 MINUTES
```

Comment:
Also I think this typepath should be tweaked slightly - right now if anyone is looking for a temporary rad immunity status effect they'll find this and use it in an inappropriate place. 

Maybe `/datum/status_effect/space_mob_radiation_immunity` or something.

Reply:
(Or alternatively keep the base type then have a subtype which includes the screen alert) 
```dm
/datum/status_effect/radiation_immunity

/datum/status_effect/radiation_immunity/space_mob
    alert_type = /whatever
```

Reply:
Yeah I can do that